src/api/routes/stats.ts

```diff
@@ -0,0 +1,46 @@
+import { TypeBoxTypeProvider } from '@fastify/type-provider-typebox';
+import { FastifyPluginAsync, FastifyPluginCallback } from 'fastify';
+import { Server } from 'http';
+import { blockParam } from '../util/helpers';
+import { Type } from '@sinclair/typebox';
+import { BlockHeightParam } from '../schemas';
+
+const IndexRoutes: FastifyPluginCallback<Record<never, never>, Server, TypeBoxTypeProvider> = (
+  fastify,
+  options,
+  done
+) => {
+  fastify.get(
```

Comment:
Can you add the `chain_tip` cache to this route? Take a look at the other endpoints to get an idea on how to do it.

Reply:
On it 👍